executor/infoschema_reader.go

```diff
@@ -2130,7 +2133,7 @@ func (e *tableStorageStatsRetriever) setDataForTableStorageStats(ctx sessionctx.
 }
 
 // dataForAnalyzeStatusHelper is a helper function which can be used in show_stats.go
-func dataForAnalyzeStatusHelper(sctx sessionctx.Context) (rows [][]types.Datum, err error) {
+func dataForAnalyzeStatusHelper(sctx sessionctx.Context, isShow bool) (rows [][]types.Datum, err error) {
```

Comment:
`isShow` seems a little ambiguous here, can we use some word more specific like `showProgress`?

Reply:
```isShow``` will be removed in the next PR. I will change the structure of the system table. it will be a big PR.